child_compassion/data/gmc_action.xml

```diff
@@ -87,11 +87,37 @@
             <field name="description">This will update the child with the major changes that happened.</field>
             <field name="auto_process">True</field>
         </record>
+        <!-- Beneficiary CDPR Messages -->
+        <record id="beneficiary_cdpr" model="gmc.action">
+            <field name="direction">in</field>
+            <field name="name">BeneficiaryCDPR</field>
+            <field name="model">compassion.child.cdpr</field>
+            <field name="description">This handles the reception of a
+                Beneficiary CDPR.</field>
```

Comment:
Please keep it on one line. You should deactivate the 79 chars length check for xml files because it does not apply for xml. Putting returns here will put the return in the description which is not wanted.